.github/workflows/images-latest.yml

```diff
@@ -11,6 +11,7 @@ on: # yamllint disable-line rule:truthy
 jobs:
   date-version:
     runs-on: ubuntu-latest
+    if: github.repository == 'dependabot/dependabot-core'
```

Comment:
Because the subsequent `push-updater-image` job has `needs: date-version`, if this causes the `date-version` job to be skipped, then the `push-updater-image` job will also get skipped.